tests/scripts/makeTestImages.sh

```diff
@@ -1,37 +1,50 @@
-#!/bin/bash -e
+#!/bin/bash
+set -ex
+
+# readable bash output
+export PS4='+${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
 
 scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # shellcheck disable=SC1090
 source "${scriptdir}/../../build/common.sh"
 
+# list of images to build
+imagesToBuild=(ceph cockroachdb cassandra edgefs nfs yugabytedb)
 
 case "${1:-}" in
     save)
+    # Storage backend to tag/save/build
+    storageBackend="$3"
+    if [ -z "$storageBackend" ]; then
+        storageBackend="all"
```

Comment:
If it's not set, why not leave it as the empty string?

Reply:
I just think it's easier to give it a proper value, it makes the code more understandable later.